packages/dioxus-translate-macro/src/lib.rs

```diff
@@ -244,6 +259,7 @@ pub fn translate_derive(input: TokenStream) -> TokenStream {
     // Generate the implementation block for `translate`
     let gen = quote! {
         impl #enum_name {
+            pub const VARIANTS: &'static [Self] = &[ #(#enum_name::#idents),* ];
```

Comment:
```suggestion
            pub const VARIANTS: &'static [Self] = &[ #(#enum_name::#idents,)* ];
```
아마도 그럴일은 없긴하겠지만, 지금 코드는 idents 가 없으면 에러 날 수 있어요.
, 가 ) 뒤에 붙으면 필드가 없어도 comma 가 붙게 되고 ) 안에 있으면 필드가 있을때만 붙어요. 참고하세요.

빌드 되는 차이가 ` [ Item1, Item2, ]` 이렇게 되게 할거냐 `[Item1, Item2]` 차이인데요. 그래서 저런 패턴은 필드가 없으면 에러나요.

Reply:
넵 알겠습니다!